docs/reST/ref/image.rst

```diff
@@ -93,18 +96,40 @@ following formats.
 
    | :sl:`save an image to disk`
    | :sg:`save(Surface, filename) -> None`
-   | :sg:`save(Surface, fileobj) -> None`
+   | :sg:`save(Surface, fileobj, namehint="") -> None`
 
    This will save your Surface as either a ``BMP``, ``TGA``, ``PNG``, or
    ``JPEG`` image. If the filename extension is unrecognized it will default to
    ``TGA``. Both ``TGA``, and ``BMP`` file formats create uncompressed files.
    You can pass a filename or a Python file-like object. For file-like object,
-   the image is saved to ``TGA`` format.
+   the image is saved to ``TGA`` format unless a namehint is used.
+
+   .. note:: To be able to save the ``JPEG`` file format to a file-like object,
+   SDL2_Image version 2.0.2 or newer is needed.
+
+   .. note:: When saving to a file-like object, it seems that for most
+   formats, it needs to be flushed after saving to be able to load from it.
 
    .. versionadded:: 1.8 Saving PNG and JPEG files.
+   .. versionchanged:: 2.0.0.dev11 The *namehint* parameter was added to make
+   it possible to save other formats than ``TGA`` to a file-like object.
 
    .. ## pygame.image.save ##
 
+.. function:: get_sdl_image_version
+
+   | :sl: `get version number of the SDL_Image library used`
+   | :sg: `get_sdl_image_version() -> None`
+   | :sg: `get_sdl_image_version() -> (major, minor, patch)`
```

Comment:
You need to delete the spaces between `:sg:` and the start of the description so it looks like so:

```
   | :sl:`get version number of the SDL_Image library used`
   | :sg:`get_sdl_image_version() -> None`
   | :sg:`get_sdl_image_version() -> (major, minor, patch)`
```